<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>合约代码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_insts"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>周期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_periods"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sub">
           <property name="text">
            <string>订阅行情</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_unsub">
           <property name="text">
            <string>退订行情</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_isHistory">
         <property name="text">
          <string>历史</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>开始时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_beginTime"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>结束时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_endTime"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget_tick">
         <column>
          <property name="text">
           <string>合约</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>周期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>最新价</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_kline">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>合约</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>周期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>高</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>低</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>收</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>1381</width>
      <height>361</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Order">
     <attribute name="title">
      <string>委托</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_Order">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>1341</width>
        <height>301</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1341</width>
        <height>0</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>13</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>报单编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>报单类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>合约</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>买卖</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开平</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>委托价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>委托量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>可撤量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>已成交量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>交易所</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>详情信息</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Account">
     <attribute name="title">
      <string>账户</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Instrument">
     <attribute name="title">
      <string>合约</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_inst">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1171</width>
        <height>291</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>11</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>品种代码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>合约</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>交易所</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>合约乘数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>最小变动价位</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>合约交易状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>多头保证金率</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>空头保证金率</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开仓手续费率</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>平仓手续费率</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>平今手续费率</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>下单</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>合约</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>买卖</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>买入</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>卖出</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>开平</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>开仓</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平仓</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>平今</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>手数</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>指定价</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>461</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>数量倍率</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>下单</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>错误信息：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_position">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>551</width>
      <height>101</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>合约</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>多空方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>总仓</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>可用仓</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>开仓均价</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_trade">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>1141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>成交编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>报单编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>合约代码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>买卖</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>开平</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成交均价</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成交量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成交类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>成交时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>交易所</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>投机套保标志</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>持仓：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>成交：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>连接行情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>连接交易</string>
    </property>
   </widget>
   <zorder>verticalLayoutWidget_3</zorder>
   <zorder>tabWidget_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_11</zorder>
   <zorder>tableWidget_position</zorder>
   <zorder>tableWidget_trade</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>tableWidget_Order</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1463</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
